aide/backend/backend_corcel.py

```diff
@@ -41,34 +50,31 @@ def query(
         **filtered_kwargs,
     }
 
-    # Measure the request time
-    t0 = time.time()
-
     # Make the API call to Corcel
     response = requests.post(
         "https://api.corcel.io/v1/text/cortext/chat", headers=headers, json=payload
     )
 
-    req_time = time.time() - t0
-
-    # Initialize output variables
-    output_content = None
-    finish_reason = None
-    info = {}
-
-    # Check if the request was successful
     if response.status_code == 200:
         # Process the Corcel API response
         data = response.json()
+
         if data and data[0].get("choices"):
             first_choice = data[0]["choices"][0]
             if "delta" in first_choice:
                 output_content = first_choice["delta"].get("content")
-                finish_reason = first_choice.get("finish_reason")
-                info = {"finish_reason": finish_reason, **payload}
-    else:
-        # Handle errors or unsuccessful responses
-        print(f"Corcel API request failed with status code {response.status_code}")
-
-    # Return the processed output along with additional information
-    return output_content, req_time, None, None, info
+            
+                # Check if the output content is a valid JSON string
+                try:
```

Comment:
check this try-catch block, it returns the same result in both cases?

Reply:
resolved